second-gen/packages/core/components/badge/Badge.base.ts

```diff
@@ -49,8 +57,9 @@ export type FixedValues = (typeof FIXED_VALUES)[number];
 
 /**
  * @element sp-badge-base
- * @slot - The text label to display in the badge.
```

Comment:
Since this doesn't have a render, should we define these only on the render layer?